### PR TITLE
Add the release talk to the docs

### PR DESCRIPTION
**Pull request by jaaberg**
Fixes [#163](https://github.com/facebook/draft-js/issues/163). Been in the loop for over a month, so I thought it was time to get it in. 

Placed it under overview section in the docs.

### CODE REVIEW

File: docs/Overview.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ Draft.js is a framework for building rich text editors in React, powered by an i
 
 Draft.js makes it easy to build any type of rich text input, whether you're just looking to support a few inline text styles or building a complex text editor for composing long-form articles.
 
+We first introduced Draft.js at the [React.js Conf](http://conf.reactjs.com/) in february:
```
Comment (hellendag) [inline docs/Overview.md]: Draft.js was introduced at [React.js Conf](http://conf.reactjs.com/) in February 2016.

**Comment by hellendag** [comment]
Just a phrasing change request. Thank you!

File: docs/Overview.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ Draft.js is a framework for building rich text editors in React, powered by an i
 
 Draft.js makes it easy to build any type of rich text input, whether you're just looking to support a few inline text styles or building a complex text editor for composing long-form articles.
 
+Draft.js was introduced at [React.js Conf](http://conf.reactjs.com/) in February 2016.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/feUYwoLhE_4" frameborder="0" allowfullscreen></iframe>
```
Comment (hellendag) [inline docs/Overview.md]: How about `width="650" height="365"`? That way it fills the whole column. :)

**Comment by jaaberg** [comment]
Thanks for the feedback, it's fixed :)

**Comment by hellendag** [comment]
Thanks!

**Comment by hellendag** [comment]
@facebook-github-bot shipit

**Comment by Daniel15** [comment]
Thank you! :smile:

**Pull request closed without merging**